python/tests/test_lowlevel.py

```diff
@@ -1832,7 +1837,7 @@ def f(mutations):
         u = ts.get_num_nodes()
         for bad_node in [u, u + 1, 2 * u]:
             self.assertRaises(_tskit.LibraryError, f, [(0.1, bad_node)])
-        for bad_pos in [-1, length, length + 1]:
+        for _bad_pos in [-1, length, length + 1]:
```

Comment:
This looks like an error, we're not actually testing the bad_pos here .

Reply:
Fixed